starter-project-web-vue2/store/sileshi.js

```diff
@@ -0,0 +1,68 @@
+import axios from 'axios'
```

Comment:
no need to import axios. It's already set up with the project. call this.$axios to use it